docs/src/user/rpc.md

```diff
@@ -0,0 +1,87 @@
+# gRPC Reference
+
+This is a reference of the Node's public RPC interface. It consists of a gRPC API which may be used to submit
+transactions and query the state of the blockchain.
+
+The gRPC service definition can be found in the Miden node's `proto`
+[directory](https://github.com/0xPolygonMiden/miden-node/tree/main/proto) in the `rpc.proto` file.
+
+<!--toc:start-->
+
+- [CheckNullifiers](#checknullifiers)
+- [CheckNullifiersByPrefix](#checknullifiersbyprefix)
+- [GetAccountDetails](#getaccountdetails)
+- [GetAccountProofs](#getaccountproofs)
+- [GetAccountStateDelta](#getaccountstatedelta)
+- [GetBlockByNumber](#getblockbynumber)
+- [GetBlockHeaderByNumber](#getblockheaderbynumber)
+- [GetNotesById](#getnotesbyid)
+- [SubmitProvenTransaction](#submitproventransaction)
+- [SyncNotes](#syncnotes)
+- [SyncState](#syncstate)
+
+<!--toc:end-->
+
+## CheckNullifiers
+
+Request proofs for a set of nullifiers.
+
+## CheckNullifiersByPrefix
+
+Request all nullifiers that match the specified prefixes. Only 16-bit prefixes are supported at this time.
+
+The prefix is used to obscure the callers interest in a specific nullifier.
```

Comment:
I think here we also can specify a block to define the lower limit for the query.